app/views/stories/_form.html.erb

```diff
@@ -0,0 +1,71 @@
+<%= form_with(model: @story, url: [@map_asset, @map_asset.stories.build], local:false) do |form| %>
+
+  <div class="modal-body">
+
+    <% if @story.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(@story.errors.count, "error") %> prohibited this story from being saved:</h2>
+        <ul>
+          <% @story.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div class="field form-group">
+      <%= form.label :name %>
+      <%= form.text_field :name, class:"form-control", readonly: true , value: @map_asset.name %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :address %>
+      <%= form.text_field :address, class:"form-control", readonly: true, value: @map_asset.address %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :category %>
+      <%= form.select :category, ["Art", "Education", "Health", "Business", "Enviroment", "Housing", "Child Care", "Food", "Industry", "Safety", "Transportation"] %>  
```

Comment:
move these options into a helper method or a constant in the model.

Reply:
Done, Created an application helper method as it will be used be two models